it/test/controllers/ContentsCheckSpec.scala

```diff
@@ -654,23 +655,33 @@ class ContentsCheckSpec extends IntegrationSpec with MockitoSugar with Matchers
     )
   )
 
-  private val userAnswers = UserAnswers("", "", Json.obj("test" -> "test"))
-    .setOrException(EndEmploymentIdPage, 1)
-    .setOrException(EmploymentDecisionPage, "company name")
-    .setOrException(EndEmploymentLatestPaymentPage, LocalDate.of(2022, 2, 2))
-    .setOrException(EndEmploymentEndDatePage, LocalDate.of(2022, 2, 2))
-    .setOrException(EndEmploymentTelephoneQuestionPage, "999")
-    .setOrException(EndEmploymentTelephoneNumberPage, "999")
-    .setOrException(AddEmploymentNamePage, "H M Revenue and Customs")
-    .setOrException(AddEmploymentPayrollNumberPage, "1234")
-    .setOrException(AddEmploymentPayrollQuestionPage, "I don't know")
-    .setOrException(AddEmploymentPayrollNumberPage, "")
-    .setOrException(AddEmploymentReceivedFirstPayPage, "Yes")
-    .setOrException(AddEmploymentStartDatePage, LocalDate.of(2022, 7, 10))
-    .setOrException(UpdateEmploymentIdPage, 1)
-    .setOrException(UpdateEmploymentNamePage, "H M Revenue and Customs")
-    .setOrException(UpdateEmploymentTelephoneQuestionPage, "No")
-    .setOrException(UpdateEmploymentDetailsPage, "Details")
+  private val userAnswers = UserAnswers(
+    "",
+    "",
+    Json.obj(
+      EndEmploymentIdPage.toString                     -> 1,
```

Comment:
These will work. But if the path on any of these objects changes in the future then they will fail. Using page.toString isn't really enough. If you look at setorexception you will see it stores the value in a concatenation of the path plus the toString.